packages/big-design/src/components/Worksheet/hooks/useStore/useStore.ts

```diff
@@ -66,3 +65,13 @@ export const createStore = () =>
     setSelectedRows: (rowIndexes) => set((state) => ({ ...state, selectedRows: rowIndexes })),
     setTableRef: (ref) => set((state) => ({ ...state, tableRef: ref })),
   }));
+
+export const useWorksheetStore = () => {
```

Comment:
🍹 Maybe rename the file to `useWorksheetStore`?

Reply:
good call! I'll rename it, thanks!